### ISSUE TITLE
Consider replacing our init.py script by cookiecutter

### ISSUE THREAD

**Issue by rkaminsk** [potassco/python-project-template#42]
https://github.com/cookiecutter/cookiecutter

**Comment by MaxOstrowski** [comment]
Very nice, I think this could be cleaner than our own init script.
I also found this
https://github.com/waynerv/cookiecutter-pypackage
which is the best python template I could find for cookiecutter so far, but it is not up to date, so I think it might make sense to make our own.

**Comment by rkaminsk** [comment]
Yes, we could build our own cookiecutter template. It would require some thought how to test the actions of a generated project. I believe this is possible but requires a bit more gitfoo.
